db/functions/base.py

```diff
@@ -396,6 +396,18 @@ def to_sa_expression(column_expr):
         return array_agg(column_expr)
 
 
+class Sum(DBFunction):
+    id = 'sum'
+    name = 'sum'
+    hints = tuple([
+        hints.aggregation,
+    ])
+
+    @staticmethod
+    def to_sa_expression(column_expr):
+        return sa_call_sql_function('sum', column_expr, return_type=PostgresType.NUMERIC)
```

Comment:
This is not accurate w.r.t. the return type, but it doesn't seem to affect the front end.

I suppose we can leave it this way since these return type declarations are going to be removed as we remove SQL Alchemy.